test/e2e/usersignup_test.go

```diff
@@ -109,6 +109,63 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestProvisionToOtherClusterWhenOneIsFull() {
+	s.T().Run("set per member clusters max number of users for both members and expect that users will be provisioned to the other member when one is full", func(t *testing.T) {
+		// given
+		var memberLimits []test.PerMemberClusterOption
+		toolchainStatus, err := s.hostAwait.WaitForToolchainStatus(wait.UntilToolchainStatusHasConditions(ToolchainStatusReadyAndUnreadyNotificationNotCreated()...))
+		require.NoError(t, err)
+		for _, m := range toolchainStatus.Status.Members {
+			if s.memberAwait.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.memberAwait.ClusterName, m.UserAccountCount+1))
+			} else if s.memberAwait2.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.memberAwait2.ClusterName, m.UserAccountCount+1))
+			}
+		}
+		require.Len(s.T(), memberLimits, 2)
+
+		userAccounts := &v1alpha1.UserAccountList{}
+		err = s.memberAwait.Client.List(context.TODO(), userAccounts)
+		require.NoError(s.T(), err)
+
+		userAccounts = &v1alpha1.UserAccountList{}
+		err = s.memberAwait2.Client.List(context.TODO(), userAccounts)
+		require.NoError(s.T(), err)
```

Comment:
what is this good for?

Reply:
Absolutely nothing! I had used them for debugging when the tests weren't working and I was too tired when I was doing the cleanup so I missed it 😆  I'll remove it.

Reply:
Done bbae633047f6053a890249136999257f3a55d6a0